tasks/rubies.yml

```diff
@@ -24,6 +24,14 @@
   when: detect_default_ruby_version.stdout == '' or
         rvm1_default_ruby_version not in detect_default_ruby_version.stdout
 
+- name: Install bundler if not installed
+  shell: 'gem list | if ! grep "^bundler " ; then {{ rvm1_install_path }}/wrappers/{{ item }}/gem install bundler ; fi'
```

Comment:
Right, sorry. Copied those lines from the #35 and didn't think about it could be incorrect. Sent a pull request with #42 to address the path fix.